deploy.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
this script depends of my current directory

Reply:
How would you remedy to this ?